pages/index.vue

```diff
@@ -13,9 +25,48 @@
 </template>
 
 <script setup lang="ts">
-const cnt = ref(0);
+import { TodoItem } from 'stores/todo'
+const todoStore = useTodoStore();
+
+const todos = computed(() => {
+  return todoStore.listOfTodos.sort((a, b) => sortMethod(a, b, "Date"));
+});
+
+// I want to use this part for sorting the list, but I can't fix the problem 
+// with being allowed of using 'sortType'... (or sortselect.value)
+const sortselect = ref(null);
+onMounted(() => {
+  const sortType: string | null = sortselect.value
+})
+</script>
+<script lang="ts">
```

Comment:
In principe hoeven we geen losse <script setup lang="ts"> en <script lang="ts"> tags te hebben. Je kan alles gewoon onder <script setup> zetten.